modules/kit-elements-defaults/module.php

```diff
@@ -11,6 +11,9 @@
 }
 
 class Module extends BaseModule {
+
+	const META_KEY = '_elementor_elements_defaults_values';
```

Comment:
```suggestion
	const META_KEY = '_elementor_elements_default_values';
```

Reply:
Good catch.
It was copy-pasted from the controller, so we'll need to fix that in the other PRs @Nevoss